### ISSUE TITLE
Feature Request: Creating and Importing XMP Sidecars

### ISSUE THREAD

**Issue by Philshappy** [FrankBijnen/ExifToolGui#507]
It is my understanding that exif tool support the creating and importing of XMP sidecar files. Many editing tools are creating sidecar files. Would it be possible to add this functionality. For example to display the sidecar data and select which data to be imported or exported.

**Comment by FrankBijnen** [comment]
Hi Phil,

The menu Export/Import supports XMP files. 
![export_import](https://github.com/user-attachments/assets/947ba620-3fcb-448e-9349-424c175b09c6)

I'm currently working on 6.3.5. and that will have more control over what gets imported.

Old:
![import_old](https://github.com/user-attachments/assets/af6177a4-11f5-4a8f-8bd5-83f1fc1583df)

New:
![import](https://github.com/user-attachments/assets/f0d64c81-473e-4b6d-92ff-c7ff44b39ca9)

I suggest you wait for the new release, couple of weeks, see what it is, and then tell me what you want.

If you want to check it out now, here's a downloadlink for the pre-rel: https://1drv.ms/f/s!AhVJC-QzM33pknOS7KwBi09lgNql?e=n7ztFX

Frank

**Comment by Philshappy** [comment]
I downloaded 6.3.5 Thanks for letting me check it out. Version 6.3.5 refers to only GPS data being imported from the XMP sidecar. I believe sidecar data could include create date and modified date, including time zone offsets. Would it be possible to import this data as well and maybe other data. I'll wait a couple of weeks and try again. Thanks so much. Great tool!

Phil

**Comment by FrankBijnen** [comment]
Hi Phil,

I think you've overlooked this.

1) Choose Export/Import Copy Metadata into JPG or Tiff files. (But copy from Single file also works)
![import1](https://github.com/user-attachments/assets/6461b2da-350f-4628-8864-78f1b1123bee)

2) Select an XMP file
![import2](https://github.com/user-attachments/assets/28c46915-8767-43e3-a09d-dc71169ad59e)

3) Select the tags to copy. This function copies standard all Tags, except the ones listed. By you can check them also.

![import3](https://github.com/user-attachments/assets/7186594b-44c8-4ee0-89e5-c7ab078f3918)

Frank

**Comment by Philshappy** [comment]
Yes I did. I updated my reply but not in time for you to get it. Yes, this is exactly what I was thinking.  I'll check back in a couple of weeks.

Thanks!

Phil

________________________________

**Comment by Philshappy** [comment]
Frank,

Will I be able to select multiple jpg files and will the tool find the matching XMP files or will I need to do 1 at a time.

Phil
________________________________

**Comment by FrankBijnen** [comment]
> Will I be able to select multiple jpg files and will the tool find the matching XMP files or will I need to do 1 at a time.

It does support multi-select. Refer to the docs: https://github.com/FrankBijnen/ExifToolGui/blob/main/Docs/ExifToolGUI_V6.md#copy-metadata-from-single-file
(The screenshots are for V6.3.4. but the functionality has not been changed)


And I would like to draw your attention to another exciting update. GUI V635 can show subfolders! All you have to do is:

- Enable show folders in the preferences
- Define a file filter with ;/s and select that


![subfolder1](https://github.com/user-attachments/assets/40b6146e-104b-4380-a5a4-c4a9e49466e1)
![subfolder2](https://github.com/user-attachments/assets/e2e8a5cc-8bc9-4eff-b9e3-17dec3fe4241)

**Comment by Philshappy** [comment]
Awsome!

Thanks So Much,

Phil
________________________________

**Comment by FrankBijnen** [comment]
@Philshappy 

HI Phil,

For your info. Just released V6.3.5. Please check out the XMP sidecar functions, and let me know.

Frank

**Comment by Philshappy** [comment]
Hi Frank,

I am sorry I won’t be able to give you any feedback on importing sidecar metadata.  When I inquired about sidecar data it was in preparation of me creating and using sidecars but I haven’t done so yet. I did test your ‘Export/Import Copy Metadata ……..’ with tag selection capability and found the options to select the metadata specifics to be very useful. Thanks so much for this capability.

Phil
